tests/test_modeling_common.py

```diff
@@ -5118,74 +4761,6 @@ def test_torch_compile_for_training(self):
         for name, param in model._orig_mod.named_parameters():
             torch.testing.assert_close(param.grad.detach().cpu(), params[name], rtol=1e-4, atol=1e-4)
 
-    @slow
-    @require_torch_gpu  # Testing cuda graphs.
-    @require_read_token
-    def test_compile_cuda_graph_time(self):
```

Comment:
Same as above (uses heavy checkpoints)

We now have a benchmarks page, much better to track performance than a test.